presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -94,13 +101,33 @@ fun AgendaScreen(
             .fillMaxSize()
             .padding(paddingValues)) {
 
-            LazyColumn(Modifier.fillMaxWidth()) {
-            /*
-            *
-            * TODO Add content here for each of the agenda items
-            *  i.e. Event, Reminders, and Tasks
-            *
-            * */
+            LazyColumn(
+                Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 8.dp),
+                verticalArrangement = Arrangement.spacedBy(16.dp)) {
+                /*
+                *
+                * TODO Add content here for each of the agenda items
+                *  i.e. Event, Reminders, and Tasks
+                *
+                * */
+                items(agendaScreenState.listOfEventDetail) { event ->
+                    AgendaCard(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(120.dp)
+                            .clip(shape = RoundedCornerShape(22.dp)),
+                        title = event.title,
+                        subtitle = event.description,
+                        dateTimeInfo = "${event.startDateTime.toZoneDateTime().formatDateTime(DATE_PATTERN)} - ${event.endDateTime.toZoneDateTime().formatDateTime(
+                            DATE_PATTERN)}",
```

Comment:
I personally prefer to pass the date info, and have the AgendaCart focus on formatting it
Since that's functionality related to the AgendaCart